examples/bpk-component-datatable/examples.tsx

```diff
@@ -74,8 +73,8 @@ const complexRows = [
 const onRowClick = (row) => alert(JSON.stringify(row));
 
 const sortFunc = (rowA, rowB) => {
-  const deskA = rowA.values.seat.desk;
-  const deskB = rowB.values.seat.desk;
+  const deskA = rowA.seat.desk;
```

Comment:
This was a bit of a bug as the row object was the one coming from `react-table` which had a lot of unnecessary properties on it. I've created a function to abstract away the `react-table` API and the row will be just the object that the consumer passes